docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -44,6 +44,10 @@ Geomaps allow you to view and customize the world map using geospatial data. You
 
 {{< figure src="/static/img/docs/geomap-panel/geomap-example-8-1-0.png" max-width="1200px" caption="Geomap panel" >}}
 
+The following video shows beginners the steps needed to create GeoMap visualizations. You'll learn the data requirements and caveats, cool customizations, preconfigured displays and much more:
```

Comment:
"Cool" is an opinion and we want to leave those out of the technical documentation. I've put in a suggestion, but if you think there's a better adjective that's more accurate to get across what's cool about them (they're popular, or super specific or something else) then please let me know.

```suggestion
The following video provides beginner steps for creating geomap visualizations. You'll learn the data requirements and caveats, special customizations, preconfigured displays and much more:
```